code/formatters.py

```diff
@@ -12,6 +12,11 @@
     ","
 )
 
+DEFAULT_SEPARATOR = ' '
```

Comment:
I'm not convinced these globals (`DEFAULT_SEPARATOR`, `SMASH_SEPARATOR`) improve readability. They're only used in one place, so this just separates their definition from where they're used.

Reply:
Have left this as is, but if you feel strongly, we can easily adjust 👍